### PR TITLE
Add refresh button portfolio

### PR DESCRIPTION
**Pull request by fabrice-adamik**

### CODE REVIEW

File: src/hooks/useAccountStateBatch.tsx [hunk 1]
```diff
@@ -11,7 +11,9 @@ export const isInAccountStateBatchCache = (
   addresses: GetAddressStateParams[]
 ) => {
   return addresses.every(({ chainId, address }) => {
-    return queryCache.find({ queryKey: ["accountState", chainId, address] });
+    return queryClientGlobal
```
Comment (fabrice-adamik) [inline src/hooks/useAccountStateBatch.tsx:14]: @henri-ly please check this part 🙏

File: src/app/portfolio/AssetsList.tsx [hunk 2]
```diff
@@ -80,22 +79,30 @@ export const AssetsList: React.FC<{
...
   setHideLowBalance,
+  refreshPositions,
 }) => {
   return (
     <>
       <Card className="lg:col-span-2">
         <CardHeader className="flex flex-row items-center justify-between">
-          <div className="flex items-center">
+          <div className="flex items-center gap-2">
+            {" "}
```
Comment (henri-ly) [inline src/app/portfolio/AssetsList.tsx:97]: A enlever si possible

File: src/hooks/useAccountStateBatch.tsx (hunk 1 above)
Comment (henri-ly) [reply src/hooks/useAccountStateBatch.tsx:14]: why do you need to do that ?
queryCache doesn't work ?

**Review by henri-ly — APPROVED** [review]

File: src/hooks/useAccountStateBatch.tsx (hunk 1 above)
Comment (fabrice-adamik) [reply src/hooks/useAccountStateBatch.tsx]: it does, I just changed it

**Pull request merged**